bolt-ktor/src/main/kotlin/com/slack/api/bolt/ktor/package.kt

```diff
@@ -33,8 +33,8 @@ suspend fun respond(call: ApplicationCall, boltResponse: Response) {
             call.response.header(header.key, value)
         }
     }
-    call.response.status(HttpStatusCode.fromValue(boltResponse.statusCode))
+    val status = HttpStatusCode.fromValue(boltResponse.statusCode)
     if (boltResponse.body != null) {
-        call.respond(TextContent(boltResponse.body, ContentType.parse(boltResponse.contentType)))
-    }
+        call.respondText(boltResponse.body, ContentType.parse(boltResponse.contentType), status)
```

Comment:
This should work but I slightly prefer using `respond` for both. What do you think?

```kotlin
    val status = HttpStatusCode.fromValue(boltResponse.statusCode)
    if (boltResponse.body != null) {
        val message = TextContent(boltResponse.body, ContentType.parse(boltResponse.contentType))
        call.respond(status, message)
    } else {
        call.respond(status)
    }
```

Reply:
Sure, I can rewrite that piece. One minute.

Reply:
Look. Is it better now?

Reply:
Thanks! LGTM